mms/tests/unit_tests/test_validate_messages.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+from mms.mxnet_model_service_error import MMSError
+from mms.utils.validators.validate_messages import ModelWorkerMessageValidators
+from mms.utils.model_server_error_codes import ModelServerErrorCodes
+import json
+
+
+def get_jsonified_str(string):
+    return json.loads(string)
+
+
+def test_validate_load_message_missing_model_path():
+    invalid_msg = get_jsonified_str('{ \"command\" : \"load\", \"modelName\" : \"some-model-name\" }')
+    with pytest.raises(MMSError) as error:
+        ModelWorkerMessageValidators.validate_load_message(invalid_msg)
+
+    assert error.value.get_code() == ModelServerErrorCodes.INVALID_LOAD_MESSAGE, 'error codes don\'t match'
+    assert error.value.get_message() == 'Load command missing \"model-path\" key'
+
+
+def test_validate_load_message_missing_model_name():
+    invalid_msg = get_jsonified_str('{ \"command\" : \"load\", \"modelPath\" : \"some-model-path\" }')
+    with pytest.raises(MMSError) as error:
+        ModelWorkerMessageValidators.validate_load_message(invalid_msg)
+
+    assert error.value.get_code() == ModelServerErrorCodes.INVALID_LOAD_MESSAGE, 'error codes don\'t match'
+    assert error.value.get_message() == 'Load command missing \"model-name\" key'
+
+
+def test_valudate_load_message_with_valid_msg():
+    valid_msg = get_jsonified_str('{ \"command\" : \"load\", \"modelPath\" : \"some-model-path\", \"modelName\" : '
+                                  '\"some-model-name\" }')
+    ModelWorkerMessageValidators.validate_load_message(valid_msg)
+
+
+def test_validate_predict_data_with_missing_request_id():
+    invalid_msg = get_jsonified_str('{\"encoding\": "None|base64|utf-8", '
+                                    '\"modelInputs\": [ {} ] } ')
+
+    with pytest.raises(MMSError) as error:
+        ModelWorkerMessageValidators.validate_predict_data(invalid_msg)
+
+    assert error.value.get_code() == ModelServerErrorCodes.INVALID_PREDICT_INPUT
+    assert error.value.get_message() == 'Predict command input missing \"request-id\" field'
+
+    valid_msg = '{ \"requestId\" : \"111-222-3333\"}'
+    ModelWorkerMessageValidators.validate_predict_data(valid_msg)
+
+
+def test_validate_predict_data_with_valid_msg():
+    valid_msg =  get_jsonified_str('{ \"requestId\": \"111-222-3333\", \"encoding\": "None|base64|utf-8", '
```

Comment:
Extra space, you should be able to see warnings in PyCharm for this.

Reply:
Done. ad207eba2c5195b6f238faf41f4231369ca9e969